pom.xml

```diff
@@ -1406,6 +1382,59 @@
           <artifactId>maven-deploy-plugin</artifactId>
           <version>2.8.2</version>
         </plugin>
+
+	<!-- This plugin's configuration is used to store Eclipse m2e settings only. -->
+	<!-- It has no influence on the Maven build itself. -->
+	<plugin>
+	  <groupId>org.eclipse.m2e</groupId>
```

Comment:
It does not change the build at all. That plugin will not get even called.

The bad: add 50 lines in pom.xml
The good: remove error warnings for folks using scala-ide (minority I guess)

Do you want to remove it?